terraform/kubernikus.tf

```diff
@@ -21,7 +21,7 @@ provider "openstack" {
 }
 
 provider "openstack" {
-  alias            = "master.na-us-1"
+  alias = "master.na-us-1"
```

Comment:
From my experience Terraform 12 doesn't like `.`s in the alias scheme. There is a validation. Unfortunately, the validation also breaks the migration that upgrades the state from 11 to 12.